File: help/eng_gb/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.js
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

//Main Function
function main() {
    document.write("Pythagorean Theorem\n" );
    document.write("Catet a \t: " );
    document.write();// print text
    a = parseFloat(prompt("a","0.0") );
    
    document.write(a + "\n");// print input 
    document.write("Catet b \t: " );
    document.write();// print text
    b = parseFloat(prompt("b","0.0") );
    
    document.write(b + "\n");// print input 
    var c = Math.sqrt( Math.pow( a , 2 ) + Math.pow( b , 2 ) ) ;
    document.write("Hypotnusa \t: " + c );
}

//JavaScript  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:56

File: help/eng_gb/exercicios_da_ficha_3/Ficha03_007/Ficha03_007.cs
// Programmer           Ant0ni0 M@ns0 - Ant0ni0 M@ns0

using System.IO;
using System;
public class Ficha03_007 {

    //Main Function
    static void Main() {
        Console.Write("Pythagorean Theorem\n" );
        Console.Write("Catet a \t: " );
        double a = double.Parse(Console.ReadLine());
        Console.Write("Catet b \t: " );
        double b = double.Parse(Console.ReadLine());
        double c = Math.Sqrt( Math.Pow( a , 2 ) + Math.Pow( b , 2 ) ) ;
        Console.Write("Hypotnusa \t: " + c );
    }

}
//C#  - Automatic translated by Algorithmi 22.05
//(c) Ant0nio M@nso 	Tue, 2022 Sep 13 21:56
